dmutils/user.py

```diff
@@ -72,7 +75,8 @@ def from_json(user_json):
             locked=user.get('locked', False),
             active=user.get('active', True),
             name=user['name'],
-            role=user['role']
+            role=user['role'],
+            user_research_opted_in=user['userResearchOptedIn']
```

Comment:
Is there a possibility that an API User response won't contain this key (tests, maybe)? It might be a good idea to initialise it to `None` as per the `supplier_foo` fields so that this is backwards-compatible.

Reply:
Hey Kat,

Thanks for the review, I have updated.